leapp/libraries/stdlib/__init__.py

```diff
@@ -175,8 +177,13 @@ def run(args, split=False, callback_raw=_console_logging_handler, callback_lineb
     api.current_logger().debug('External command has started: {0}'.format(str(args)))
     _id = str(uuid.uuid4())
     result = None
+    _path = env.get('PATH', None) if env else None
```

Comment:
Nit: `_path = (env or {}).get('PATH')` should give you the same result

Reply:
updated